docs/howtos/another_kolibri_instance.md

```diff
@@ -0,0 +1,55 @@
+## Running Another Kolibri Instance Alongside the Development Server
+
+This guide will walk you through the process of setting up and running another instance of Kolibri alongside your development server using the `pex` executable.
+
+## Introduction
+
+As Kolibri's features continue to expand into remote content browsing, it's often necessary to test and experiment with another Kolibri instance running alongside your development server. One effective approach is to use the `pex` executable. This workflow is straightforward and can be employed independently of ZeroTier or even internet network access. Documenting these steps will provide a handy reference for both new team members and contributors.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A recent `pex` executable of Kolibri. You can download this from the Build Assets page of a workflow build.

```

Comment:
Generally, it's good to have prerequisites section, but in this particular case it felt a bit confusing to me as below in "Steps", we describe how to get the executable. I think it's a good idea to provide guidance on downloading so I'd suggest removing it from "Prerequisites" and keep only as part of "Steps".

Reply:
Agreed with  points made by @MisRob, and since there are no other _prerequisites_, we can remove this whole ## section.